lib/Jsrt/ChakraCommon.h

```diff
@@ -1040,9 +1040,9 @@ typedef unsigned char* ChakraBytePtr;
     /// </returns>
     CHAKRA_API
         JsParseScript(
-            _In_z_ const wchar_t *script,
+            _In_z_ const CSTR_TYPE *script,
```

Comment:
Not sure if this affects MSDN documentation.

Are these APIs Win32 only? If so keeping wchar_t looks fine? + @digitalinfinity